src/ngraph/runtime/reference/atan2.hpp

```diff
@@ -0,0 +1,38 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include <cmath>
+#include <cstddef>
+
+namespace ngraph
+{
+    namespace runtime
+    {
+        namespace reference
+        {
+            template <typename X, typename Y, typename Z>
+            void atan2(const X* px, const Y* py, Z* pout, size_t count)
+            {
+                for (size_t i = 0; i < count; i++)
+                {
+                    *pout++ = static_cast<Z>(std::atan2(*px++, *py++));
```

Comment:
The constructor has `(y,x)`
std's atan2 is also `atan2( float y, float x )`
but this function has `const X* px, const Y* py`

Maybe we can reverse order?

Reply:
Missed one.